### PR TITLE
Updated README to build the PostgreSQL cartridge + bug fix

### PR DESCRIPTION
**Pull request by ptosco**
- renamed `Code/PgSQL/rdkit/README` to `Code/PgSQL/rdkit/README.md` such that markdown can be used
- updated `Code/PgSQL/rdkit/README.md` with updated instructions how to build and install the PostgreSQL cartridge on all supported OSs
- fixed an issue with the PgSQL install script on Windows where the path to the `update_sql` directory was incorrect

### CODE REVIEW

**Comment by ptosco** [comment]
@greglandrum The macOS failure here is due to the usual `pySimDivPickers` failing and is unrelated to this PR.

**Review by greglandrum — APPROVED** [review]
I'm not sure how robust the details of all the commands are in terms of inter-system variability, but this is clearly an expert user topic and there's only so much we can do.

**Comment by greglandrum** [comment]
@ptosco I'm willing to merge this as is, but if you have thoughts about the "how to restart the server" point, I thought I'd give you some time.

File: Code/PgSQL/rdkit/README.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+administrator privileges, stop the PostgreSQL service, run
+/home/build/build/rdkit/Code/PgSQL/rdkit/pgsql_install.sh
+to install the PostgreSQL RDKit cartridge, then start again
+the PostgreSQL service.
+=====================================================================
+```
+If you can gain sudo privileges, you can install the cartridge
+as shown below.
+```
+$ sudo /etc/init.d/postgresql stop
```
[the remark this answers is not in the document]
Comment (ptosco) [reply Code/PgSQL/rdkit/README.md]: Yeah, WSL2 still uses the old `SysV` lingo, but that's quite an exception these days. I updated the README.

**Comment by ptosco** [comment]
> I'm not sure how robust the details of all the commands are in terms of inter-system variability, but this is clearly an expert user topic and there's only so much we can do.

Exactly. This is more intended as examples of things that can go wrong and possible solutions than as a recipe for all needs.

**Pull request merged**